src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -5775,6 +5775,28 @@ static object F()
             await AssertFormatAsync(expectedCode, code);
         }
 
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task SpacingInNullCheckedParameter()
```

Comment:
Don't forget to also test/handle the syntax normalize (SyntaxNormalizerTests.cs) which generally needs similar fixing.

Reply:
I didn't see a similar test in SyntaxNormalizerTests for the `!` operator. Didn't we have to do something similar with that for formatting?

Reply:
It wasn't clear to me what needs to be done here.